cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/worker/RunnableTaskLauncher.java

```diff
@@ -16,36 +16,76 @@
 
 package io.cdap.cdap.internal.app.worker;
 
+import com.google.inject.Guice;
+import com.google.inject.Injector;
 import io.cdap.cdap.common.conf.CConfiguration;
-import io.cdap.cdap.common.internal.worker.RunnableTask;
-import io.cdap.cdap.common.internal.worker.RunnableTaskContext;
+import io.cdap.cdap.common.id.Id;
+import io.cdap.cdap.internal.app.runtime.artifact.ArtifactDetail;
+import io.cdap.cdap.internal.app.runtime.artifact.ArtifactRepository;
+import io.cdap.cdap.internal.app.runtime.artifact.ArtifactRepositoryReader;
+import io.cdap.cdap.internal.worker.api.RunnableTask;
+import io.cdap.cdap.internal.worker.api.RunnableTaskContext;
+import io.cdap.cdap.internal.worker.api.RunnableTaskRequest;
+import io.cdap.cdap.security.impersonation.EntityImpersonator;
+import io.cdap.cdap.security.impersonation.Impersonator;
+import org.apache.hadoop.conf.Configuration;
 
 /**
  * RunnableTaskLauncher launches a {@link RunnableTask} by loading its class and calling its run method.
  */
 public class RunnableTaskLauncher {
-  private final CConfiguration cConf;
+  private final CConfiguration cConfig;
+  private final Configuration hConf;
+  private final ArtifactRepositoryReader artifactRepositoryReader;
+  private final ArtifactRepository artifactRepository;
+  private final Impersonator impersonator;
 
-  public RunnableTaskLauncher(CConfiguration cConf) {
-    this.cConf = cConf;
+  public RunnableTaskLauncher(CConfiguration cConfig,
```

Comment:
call it cConf to be consistent with the rest of the code base.

Reply:
Done.